plugins/database/mysql/mysql.go

```diff
@@ -197,6 +182,32 @@ func (m *MySQL) CreateUser(ctx context.Context, statements dbplugin.Statements,
 	return username, password, nil
 }
 
+func executeCreateUserStmt(ctx context.Context, tx *sql.Tx, query string) (shouldContinue bool, err error) {
```

Comment:
Same comment here.

Reply:
On this one, if I move it back up to being written inline, the `defer` statement will continue to run within a loop. I'm fine with that, just want to make sure you are too.

Reply:
I'd just put an explicit `stmt.Close` before each `return` statement, there's only two. Then there's no issue.

Reply:
K cool! Will do!